src/signals/incident-management/containers/IncidentOverviewPage/saga.js

```diff
@@ -40,9 +42,47 @@ export function* openIncident(action) {
   yield put(push(navigateUrl));
 }
 
+export function* getFilters() {
+  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/me/filters/`;
+
+  try {
+    const result = yield authCall(requestURL);
+
+    yield put(getFiltersSuccess(result.results));
+  } catch (error) {
+    yield put(getFiltersFailed(error));
+  }
+}
+
+export function* removeFilter(action) {
+  const id = action.payload;
+  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/me/filters/${id}`;
+
+  try {
+    yield authDeleteCall(requestURL);
```

Comment:
I'm seeing a lot of authentication/authorization checks, nice. But there doesn't seem to be a flow when auth fails.

Reply:
we hebben op de backlog een story voor betere error flows. die moet daar ook bij.